angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc

```diff
@@ -18,6 +18,23 @@
          "altMaxUrl":false
       },
       {
+         "nodeId":"u23l1n234",
+         "title":"/r/uwmadison",
+         "description":"The UW-Madison subreddit. We do not monitor this feed.",
+         "url":"https://www.reddit.com/r/uwmadison",
+         "iconUrl":null,
+         "faIcon":"fa-reddit",
+         "fname":"uwmadison-sub-reddit",
+         "target": "_blank",
+         "widgetURL": "/web/api/uwmadisonreddit.json",
+         "widgetType":"generic",
+         "widgetTemplate":"<div style='max-height: 210px; overflow-y: scroll' ng-init='lim=5'> <ul class='widget-list' style='padding : 0'> <li ng-repeat='item in content.data.children | limitTo:lim'> <a href='{{item.data.url}}' target='_blank'> <p class='bold'>{{item.data.title}}<span class='right'>score: {{item.data.score}}</span></p> <p>{{item.data.selftext}}</p> </a> </li> <li class='center' ng-hide='content.data.children.length <= lim' ng-click='lim=lim+5'><button class='btn btn-flat center'>Load More</button></li></ul></div><a class='btn btn-default launch-app-button' href='https://www.reddit.com/r/uwmadison' target='_blank'>Launch UW subreddit</a>",
```

Comment:
This incantation

```
style='max-height: 210px; overflow-y: scroll'
```

looks useful and re-usable for other Widgets in this style.  Worth pulling out into some `css`?